### ISSUE TITLE
Leading escape sequence

### ISSUE THREAD

**Issue by alexprengere** [opentraveldata/geobases#16]
The `import readline` statement causes a leading escape sequence in `stdout`. This causes a bug in quiet mode when using the CLI.

**Comment by alexprengere** [comment]
Added 997e3a11c3dda4fa25b87b33f6d207c4d8380486 then 48d47280502d4d0993b32982a01f2f36816af289 on `develop` branch to fix this problem in next release.

**Comment by alexprengere** [comment]
This problem is bugging me. The fix I wrote earlier does not work, despite redirecting `stdout` and `stderr` to `/dev/null`. I opened a [question on stackoverflow](http://stackoverflow.com/questions/15760712/python-readline-module-prints-escape-character-during-import) about this.

In the meantime, I wrote another fix 12967c85d8b4b749d372bf906db8efe3f3e72af0 based on [this](http://reinout.vanrees.org/weblog/2009/08/14/readline-invisible-character-hack.html), but I am still curious about this issue.
